inc/plugin.class.php

```diff
@@ -347,14 +348,10 @@ function checkStates() {
    static function isAllPluginsCSRFCompliant() {
```

Comment:
This method is not used (only use of it is commented).

Reply:
Indeed, even if it's out of the current PR scope

Reply:
https://github.com/glpi-project/glpi/pull/4460